lib/hardwareWalletUtilities.js

```diff
@@ -0,0 +1,103 @@
+import { Script, Outpoint, TX, MTX, Coin } from 'bcoin';
```

Comment:
A note for the future, I imagine these might be useful utilities to have in `bpanel-utils`. Just from a security standpoint too it would be good if plugin devs could (and were encouraged to) use our utilities to communicating with hardware. I could be wrong but these utilities seem pretty general and could be used in other applications right?